packages/react/src/components/Table/TableHead/TableHead.test.jsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import { mount } from 'enzyme';
```

Comment:
Nice work! Do we want to remove enzyme this go around or open an issue to tackle it later?

Reply:
Good question. There are a lot of enzyme tests for this component. I think we should do that in a separate move, the primary objective was the coverage increase.

Reply:
I tend to rewrite the tests when it feels feasible in relation to the effort of increasing the coverage

Reply:
agreed. Just wanted to check.